src/nwm_routing/setup.cfg

```diff
@@ -0,0 +1,20 @@
+[metadata]
+name=nwm_routing
+author=James Halgren
+author_email=james.halgren@noaa.gov
+
+[options]
+package_dir=
+  =src
+packages=find:
+python_requires=>=3.6
+install_requires=
+  numpy
+  Cython
+  pandas
```

Comment:
IIRC, James ran into some issues using an older version of pandas. I believe the constraint here is pandas >= 1.2

Reply:
Hmm....using python 3.6.8, pandas 1.1.5 is the latest available version.  We should really check into this in more detail if there is a bug or something that doesn't work with this version.  I haven't seen it yet, but doesn't mean I have hit it...For now, I'm going to revert the requirement to 1.1.5, we can adjust if needed, but that will mean a python 3.7.1 minimum if we want to use pandas 1.2 as the minimum.

Reply:
So far, using the latest version of pandas on 3.6.8 (i.e. pandas 1.1.5) does not trigger the anomalous behavior so that should be a sufficient version requirement for now.